app/instance.py

```diff
@@ -251,10 +240,10 @@ def update_sent_state(sender=None, headers=None, **kwargs):
 scheduler = BackgroundScheduler(timezone=utc)
 # scheduler.add_job(send_mail_to_expired_orders, 'interval', hours=5)
 # scheduler.add_job(empty_trash, 'cron', hour=5, minute=30)
-if app.config['ENABLE_ELASTICSEARCH']:
-    from app.views.elastic_cron_helpers import sync_events_elasticsearch, cron_rebuild_events_elasticsearch
-    scheduler.add_job(sync_events_elasticsearch, 'interval', minutes=60)
-    scheduler.add_job(cron_rebuild_events_elasticsearch, 'cron', day=7)
+# if app.config['ENABLE_ELASTICSEARCH']:
+#     from app.views.elastic_cron_helpers import sync_events_elasticsearch, cron_rebuild_events_elasticsearch
+#     scheduler.add_job(sync_events_elasticsearch, 'interval', minutes=60)
+#     scheduler.add_job(cron_rebuild_events_elasticsearch, 'cron', day=7)
```

Comment:
Delete the code, don't comment

This is a git project, not a college project shared in pen drives